docs/posts/annotations.md

```diff
@@ -13,16 +13,15 @@ We are _piloting_ a new feature in cellxgene that enables users to create and ed
 
 To get started, run:
 ```
-cellxgene launch mydata.h5ad --experimental-annotations
+cellxgene launch mydata.h5ad
 ```
 
 To preserve data provenance, **`cellxgene` does not alter the input h5ad file**.  Rather, newly-created annotations are saved in a specified CSV file:  
 ```
 annotations-directory/name-########.csv
 ```
 
-- The default `annotations-directory` is  your current working directory (i.e., the directory you were in when you started cellxgene).
-
+- The default `annotations-directory` is the directory where `mydata.h5ad` is saved; if we can't find that directory (e.g., you're loading an h5ad from a URL), it falls back to your current working directory (i.e., the directory you were in when you started cellxgene).
```

Comment:
I would also point out that it will default to CWD if it can't write to the location of the H5AD.

Eg., `(e.g., you're loading an h5ad from a URL, or the location of the h5ad is not writable)`